metis/lib/metis.rb

```diff
@@ -13,7 +13,6 @@ class Metis
   def setup_db
     @db = Sequel.connect(config(:db))
     @db.extension :connection_validator
-    @db.pool.connection_validation_timeout = -1
```

Comment:
@graft , I took this line out because, per my reading of [the documentation](https://sequel.jeremyevans.net/rdoc-plugins/files/lib/sequel/extensions/connection_validator_rb.html), using this setting may be less performant, in that a query must be executed each time to validate a connection in the pool? It also looks like we're not using the suggested coarse connection checkout in middleware. Testing this locally I can't really tell a difference in performance with this setting and without -- curious if you recall why you disabled the connection caching, originally? With the increased usage of Metis for ETLs and uploads, I feel like connections shouldn't expire, naturally, and forcing a validation each time may be unnecessary ...

Reply:
Hmm.  I do think it's wise to leave this, tbh.  I don't think it effects our performance significantly and it could help reliability of the pool.  I just don't think this is causing a significant performance issue.